### DISCUSSION TITLE
Consider removing WinBox message from 86box Builds

### DISCUSSION THREAD

**Discussion by MattBytesRetro** [86Box/86Box#4837] [Ideas]
For a while now, 86box has presented a message when using WinBox, that it's no longer supported because 'development stopped in 2022' and suggests going to the 86box site for alternatives.

They are:
[86BoxManager] - No Updates also since mid 2022
[86BoxManagerX] - No Updates since mid 2023
[linbox-qt5] - Linux Only
[MacBox] - The best maintained, but Mac Only.

Unless there are other reasons, I am not aware of, (I still use WinBox with current dev builds, without noticeable issues), I feel this message should have a checkbox for "Do not show again", until something better does come along. At least for me, WinBox still has the best GUI and feature set for Windows users.

If there's other reasons, I am unaware of or am missing, then my apologies in advance.

**Comment by jadasse** [comment]
I agree with this

**Comment by nanitaro** [comment]
One of the developers said
>https://www.vogons.org/viewtopic.php?t=101979
>Easy - you use a different manager. WinBox messes directly with 86Box's internals and due to the fact it hasn't been updated in over 2 years now, it's been causing numerous problems, misleading people to believe they're actually bugs in 86Box itself.

Winbox should be discontinued as it adversely affects the operation of 86box.
It is an action that annoys developers.

Unlike Winbox, 86BoxManager is a simple launcher and is most compatible with the latest 86box.
It is a simple launcher and has few glitches, so it does not need to be updated.
There is no need for Winbox as 86box itself has an easy-to-use configuration UI.

**Reply by MattBytesRetro** [reply]
Out of curiosity what are some issues winbox causes?  Ty

**Reply by OBattler** [reply]
WinBox directly messes with 86Box menu's, etc., which can result in unpredictable behavior when 86Box is not exactly as it was at the time of WinBox's last update.

**Comment by Alexrowland473** [comment]
I agree with this
